js68/SOURCES/spidermonkey_checks_disable.patch

```diff
@@ -0,0 +1,10 @@
+--- a/config/run_spidermonkey_checks.py
++++ b/config/run_spidermonkey_checks.py
+@@ -11,5 +11,5 @@
+     for script in scripts:
+         retcode = subprocess.call(
+             [sys.executable, script], cwd=buildconfig.topsrcdir)
+-        if retcode != 0:
+-            raise Exception(script + " failed")
++        #if retcode != 0:
++        #    raise Exception(script + " failed")
```

Comment:
This one seems a little suspect, are we sure it's still best practice to skip these tests altogether?

Reply:
I have no idea, this was part of a set of patches that have been used to build the Fedora RPMs since what I believe is basically forever, so it's entirely possible that it's now obsolete.